src/ui/SmallVideoPreview/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { ReactElement } from 'react'
+import './styles.css'
+import SmallVideoDetails from '../SmallVideoPreview/SmallVideoDetails'
+
+interface VideoPreviewProps {
```

Comment:
Props name should match component name
```suggestion
interface SmallVideoPreviewProps {
```